app/models/card.rb

```diff
@@ -1,4 +1,7 @@
 class Card < ActiveRecord::Base
+  belongs_to  :user
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.